packages/sdks/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -350,7 +350,7 @@ class BaseDescopeWc extends BaseClass {
   }
 
   async #handleDebugMode({ isDebug }) {
-    if (isDebug) {
+    if (isDebug && process.env.NODE_ENV === 'development') {
```

Comment:
Need to doc somehow
And log a warning if debug is true but env is not set to development
```suggestion
    if (isDebug && process.env.NODE_ENV !== 'production') {
```